traits/src/oracle.rs

```diff
@@ -69,6 +73,17 @@ impl OraclePeriod {
 		use OraclePeriod::*;
 		&[Short, TenMinutes, Hour, Day, Week]
 	}
+
+	pub const fn as_period(&self) -> u64 {
```

Comment:
Nice but... people might be mistaken if they change this, oracle periods will change.. We should probably write test to see if it matches smoothing factors down.